config/routes.rb

```diff
@@ -7,5 +7,7 @@
   patch '/profile', to: 'users#update'
   get '/profile', to: 'users#profile', as: 'user_profile'
   patch '/update_profile_picture', to: 'users#update_profile_picture', as: 'update_profile_picture'
+  get 'used-cars/sell', to: 'used_cars#index'
+  get '/post-ad' , to: 'used_cars#new_ad'
```

Comment:
This route should be `used-cars/sell, to: 'used_cars#post-ad'` the way we have here on actual domain => https://www.pakwheels.com/used-cars/sell/post-ad